Helper/Cart.php

```diff
@@ -1023,7 +1027,9 @@ protected function createImmutableQuote($quote)
     {
         // assign origin session type flag to the parent quote
         // to be replicated to the immutable quote with the other data
-        $quote->setBoltIsBackendOrder( (int) $this->isBackendSession() );
+        if ($this->isBackendSession()) {
+            $quote->setBoltCheckoutType(self::BOLT_CHECKOUT_TYPE_BACKOFFICE);
+        }
```

Comment:
What happens if bolt checkout type is left unset?

Reply:
@njfaries The default value of the field in DB is 1, corresponding to BOLT_CHECKOUT_TYPE_MULTISTEP.